# day15

## [注解]()

## [JUnit]()

## [异常](https://github.com/Charon-33/Learning_JavaSE/tree/day15/src/com/atguigu/exception)
1. [异常的概述](https://github.com/Charon-33/Learning_JavaSE/tree/day15/src/com/atguigu/exception/TestException.java)
2. [异常的类型](https://github.com/Charon-33/Learning_JavaSE/tree/day15/src/com/atguigu/exception/TestExceptionType.java)
3. [异常处理](https://github.com/Charon-33/Learning_JavaSE/tree/day15/src/com/atguigu/exception/TestTryCatch.java)
4. [获取异常对象信息](https://github.com/Charon-33/Learning_JavaSE/tree/day15/src/com/atguigu/exception/TestExceptionInfo.java)
5. [finally块](https://github.com/Charon-33/Learning_JavaSE/tree/day15/src/com/atguigu/exception/TestFinally.java)
6. finally和return：[1](https://github.com/Charon-33/Learning_JavaSE/tree/day15/src/com/atguigu/exception/TestFinallyReturn1.java) [2](https://github.com/Charon-33/Learning_JavaSE/tree/day15/src/com/atguigu/exception/TestFinallyReturn2.java) [3](https://github.com/Charon-33/Learning_JavaSE/tree/day15/src/com/atguigu/exception/TestFinallyReturn3.java) [4](https://github.com/Charon-33/Learning_JavaSE/tree/day15/src/com/atguigu/exception/TestFinallyReturn4.java)
7. 关键字throws：[1](https://github.com/Charon-33/Learning_JavaSE/tree/day15/src/com/atguigu/exception/TestThrows.java) [2](https://github.com/Charon-33/Learning_JavaSE/tree/day15/src/com/atguigu/exception/TestThrows2.java)
8. [关键字throws对重写方法的要求](https://github.com/Charon-33/Learning_JavaSE/tree/day15/src/com/atguigu/exception/TestOverride.java)
9. [关键字throw](https://github.com/Charon-33/Learning_JavaSE/tree/day15/src/com/atguigu/exception/TestThrow.java)